README.md

```diff
@@ -117,6 +117,8 @@ The following are instructions for running our tests:
     `python setup.py install`
 4. Run the following command to test the plugin in all versions of python we support:
     `tox`
+    
+Once you have all required package installed, you can run tests locally with `python -m unittest -v test.[test file name].[test case name].[individual test name]`.
```

Comment:
* Should be `packages`,
* `-v`, which stands for `verbose`, is optional,
* Shouldn't it be `class name` instead of `test case name`?
* Also, this is to run individual tests. Users will most likely be interested in running all tests which can be accomplished by `python -m unittest`